prov/gni/src/gnix_util.c

```diff
@@ -161,3 +161,29 @@ int gnixu_get_rdma_credentials(void *addr, uint8_t *ptag, uint32_t *cookie)
 	}
 	return ret;
 }
+
+#define NUM_GNI_RC (GNI_RC_ERROR_NOMEM+1)
+static int gnix_rc_table[NUM_GNI_RC] = {
+	FI_SUCCESS, /* GNI_RC_SUCCESS = 0 */
```

Comment:
Good idea.  The codes are in `gni_pub.h` and should be available.  I'll make that change.